src/index.html

```diff
@@ -4,9 +4,43 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/blocks/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="container">
```

Comment:
Would you happen to really need this container? you can just set all flex properties to the calendar directly and it will work the same way, with fewer tags